### DISCUSSION TITLE
Perfectly Working Server Unable to use CLi

### DISCUSSION THREAD

**Discussion by reatherly** [gogs/gogs#7742] [Q&A]
I have my server up and running perfectly the way I want it. With one exception. It will not allow the use of CLi for any repository actions. I get a 403 Error everytime. but I can make changes and commit using the WebUI. I have checked permission, settings, configurations and everything else to no avail. Any help would be appreciated. Thank you.

**Comment by annpocoyo** [comment]
Can you please send the logs and exact output when you run the command?

**Comment by reatherly** [comment]
Oddly enough I just tried to reproduce the result and was unable to do so.
It worked flawlessly this time. But I believe the original error was HTTP
Error 4040 Not Found.

**Reply by annpocoyo** [reply]
Well good for you.
